resources/js/contest-judge-results/header.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+import BasicSelectOptions from 'components/basic-select-options';
+import UserLink from 'components/user-link';
+import ValueDisplay from 'components/value-display';
+import ContestEntryJson from 'interfaces/contest-entry-json';
+import ContestJson from 'interfaces/contest-json';
+import SelectOptionJson from 'interfaces/select-option-json';
+import * as React from 'react';
+import { trans } from 'utils/lang';
+
+interface Props {
+  contest: ContestJson;
+  entries: ContestEntryJson[];
+  entry: ContestEntryJson;
+}
+
+export default class Header extends React.PureComponent<Props> {
+  private get selectOptions(): SelectOptionJson[] {
+    const ret = [];
+
+    for (const entry of this.props.entries) {
+      ret.push(this.entryToSelectOption(entry));
+    }
+
+    return ret;
+  }
+
+  render() {
+    const { contest, entry } = this.props;
+
+    const score = entry.results?.votes;
+    const maxScore = (contest.max_judging_score ?? 0) * (entry.judge_votes?.length ?? 0);
```

Comment:
well max_judging_score is sum of scoring_category values so technically yes 🤨

all judges should be required to vote on all entries for things to make sense though so it shouldn't differ per-entry of one contest, ended up moving this part to backend anyway since it probably makes way more sense.